db-engine/src/main/java/com/splicemachine/db/catalog/types/IndexDescriptorImpl.java

```diff
@@ -269,20 +322,35 @@ else if (isUniqueWithDuplicateNulls)
 	public void readExternal(ObjectInput in) throws IOException, ClassNotFoundException {
         FormatableHashtable fh = (FormatableHashtable) in.readObject();
         isUnique = fh.getBoolean("isUnique");
-        int bcpLength = fh.getInt("keyLength");
-        baseColumnPositions = new int[bcpLength];
-        isAscending = new boolean[bcpLength];
-        for (int i = 0; i < bcpLength; i++) {
-            baseColumnPositions[i] = fh.getInt("bcp" + i);
-            isAscending[i] = fh.getBoolean("isAsc" + i);
-        }
+        int keyLength = fh.getInt("keyLength");
+		isAscending = new boolean[keyLength];
```

Comment:
is that tab/spaces mixed?

Reply:
Yes I also found my Intellij is confused about mixing tabs and spaces after pushing these commits. I try to fix them in the following commits.